### PR TITLE
Feat  exp comp

### PR DESCRIPTION
**Pull request by austinmargarone**
@brandonetter

### CODE REVIEW

File: components/WorkExp.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import WorkExpCard from "./WorkExpCard";
+import Image from "next/image";
+
+type Props = {};
+
+const WorkExp = (props: Props) => {
```
Comment (brandonetter) [inline components/WorkExp.tsx:7]: Are you planning on passing props into `WorkExp`?

**Review by brandonetter — CHANGES_REQUESTED** [review]
Fix conflicts, and I have one question:

File: components/WorkExp.tsx (hunk 1 above)
Comment (austinmargarone) [reply components/WorkExp.tsx:7]: No. That's there because I ran tsrafce and forgot to remove it.

**Comment by brandonetter** [comment]
👍

**Pull request merged**